src/store-api/src/manifest/storage.rs

```diff
@@ -46,6 +46,9 @@ pub trait ManifestLogStorage {
         keep_last_checkpoint: bool,
     ) -> Result<usize, Self::Error>;
 
+    /// Delete all logs and checkpoints, and remove the manifest directory.
```

Comment:
We should mention that we assume the last log contains a `Remove` action
```suggestion
    /// Delete all logs and checkpoints, and remove the manifest directory.
    /// This method assumes the last log file contains a remove meta action.
```

But, is it possible to pass the version of the file with the remove action to this method? So we don't such an assumption.
```rust
async fn delete_all(&self, remove_action_version: ManifestVersion) -> Result<(), Self::Error>;
```

Since we could get the manifest version when we write the `RegionRemove` action.